docs/specification/appendix.md

```diff
@@ -67,6 +71,12 @@ Schema `encoding` may only be omitted for self-describing message encodings such
 - `encoding`: `flatbuffer`
 - `data`: A [reflection.Schema](https://github.com/google/flatbuffers/blob/master/reflection/reflection.fbs) flatbuffer describing the parsed schema encoded as a binary flatbuffer. This can be generated by running `flatc -b --schema` on the fbs file.
 
+### capnproto
+
+- `name`: Fully qualified name: `[schema display name]:Name`. For example, the struct `Bar` in schema file with compilation path `path/to/files/schema.capnp` has the default qualified name `path/to/files/schema.capnp:Bar`. `[schema display name]` can be overridden to limited effect via `--src-prefix`.
```

Comment:
From the docs it sounds like this qualified name may not be unique. Do you have any other ideas for how to identify nodes in the schema?

Also discussing at https://github.com/capnproto/capnproto/issues/673#issuecomment-1476783025

I found some examples of dynamic schema usage here:
- https://capnproto.org/cxx.html
- https://groups.google.com/g/capnproto/c/sNEVK0WKa7o/m/7gFCFmEpvcEJ
- https://github.com/capnproto/capnproto/blob/master/c%2B%2B/src/capnp/schema-loader.h#L33-L34

But I don't see any discussion here of how to use a human-readable name to identify a schema in a CodeGeneratorRequest.

Reply:
> From the docs it sounds like this qualified name may not be unique.

I think this is an unlikely edge case. In most cases, the names will be unique. We will throw an error in `capnp-reflection` library or mcap CLI when loading two schemas with the same `displayName`. 

If users report this to us, we can suggest fixes.

The specific scenario where displayName may not be unique that I can think of is:
```bash
> ls tree
schemas
|-- my_schema.capnp
|-- subdir
     |-- my_schema.capnp
       
> cat my_schema.capnp

import "../dep.capnp"

struct MyStruct {
  my_dep_struct: "myschema.capnp:MyStruct",
}

> capnp compile --src-prefix=my_schema ./subdir/my_schema.capnp | capnp convert binary:json CodeGeneratorRequest
..
nodes: [
  .. 
  id: 1234,
  displayName: "my_schema.capnp:MyStruct"
  ..
  id: 5432,
  displayName: "my_schema.capnp:MyStruct"
  ..
 
```

 In this case, the error is totally fixable if the user omits `--src-prefix`. Naming two files the same name is a bad practice and unlikely in my opinion.

Reply:
The alternative is to use the unique schema `Id`, however, this is not human readable and not present in the schema file (it requires the capnp compiler to show)